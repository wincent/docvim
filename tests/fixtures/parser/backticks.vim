""
" We can use `backticks` as well.
"
" Just remember to `git commit` at the end of it.
