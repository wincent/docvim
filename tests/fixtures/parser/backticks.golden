Project
  [ Project
      [ Unit
          [ DocBlock
              [ Paragraph
                  [ Plaintext "We"
                  , Whitespace
                  , Plaintext "can"
                  , Whitespace
                  , Plaintext "use"
                  , Whitespace
                  , Code "backticks"
                  , Whitespace
                  , Plaintext "as"
                  , Whitespace
                  , Plaintext "well."
                  ]
              , Paragraph
                  [ Plaintext "Just"
                  , Whitespace
                  , Plaintext "remember"
                  , Whitespace
                  , Plaintext "to"
                  , Whitespace
                  , Code "git commit"
                  , Whitespace
                  , Plaintext "at"
                  , Whitespace
                  , Plaintext "the"
                  , Whitespace
                  , Plaintext "end"
                  , Whitespace
                  , Plaintext "of"
                  , Whitespace
                  , Plaintext "it."
                  ]
              , Paragraph [ Code "backticks in first column work too" ]
              ]
          ]
      ]
  ]
